stable/external-dns/values.yaml

```diff
@@ -280,6 +280,9 @@ annotationFilter: ""
 ## When enabled, prints DNS record changes rather than actually performing them
 ##
 dryRun: false
+## When enabled, triggers run loop on create/update/delete events (optional, in addition of regular interval)
+##
+events: false
```

Comment:
Please, add it to the `values-production.yaml` too.

Reply:
done